javalib/src/main/scala/java/util/ArrayList.scala

```diff
@@ -44,50 +71,134 @@ class ArrayList[E] private (private[ArrayList] val inner: js.Array[E])
   }
```

Comment:
We should implement `ensureCapacity` and `trimToSize` for Wasm. They're meaningful now.